LDAR_Sim/model_code/methods/deployment/base.py

```diff
@@ -0,0 +1,53 @@
+class sched_crew:
+    def get_work_hours(self):
+        """ Get hours in day the crew is able to work
+        """
+        if self.config['consider_daylight']:
+            daylight_hours = self.state['daylight'].get_daylight(
+                self.state['t'].current_timestep)
+            if daylight_hours <= self.config['max_workday']:
+                self.work_hours = daylight_hours
+            elif daylight_hours > self.config['max_workday']:
+                self.work_hours = self.config['max_workday']
+        elif not self.config['consider_daylight']:
+            self.work_hours = self.config['max_workday']
+
+        if self.work_hours < 24 and self.work_hours != 0:
+            self.start_hour = (24 - self.work_hours) / 2
+            self.end_hour = self.start_hour + self.work_hours
+        else:
+            print(
+                'Unreasonable number of work hours specified for crew ' +
+                str(self.crewstate['id']))
+
+        self.allowed_end_time = self.state['t'].current_date.replace(
+            hour=int(self.end_hour), minute=0, second=0)
+
+
+class sched_company:
```

Comment:
Classes should use CamelCase convention. Is a class needed here? If so, I wonder if there's a more descriptive name, like CompanySchedule or CompanyScheduling or something

Reply:
Fixed